configs/components/rubygem-nokogiri.rb

```diff
@@ -1,16 +1,31 @@
 component "rubygem-nokogiri" do |pkg, settings, platform|
-  pkg.version "1.6.6.2"
-  pkg.md5sum "fc9f91534bf93d57b84f625b55732a7c"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/nokogiri-#{pkg.get_version}.gem"
+  pkg.version "1.6.7.2"
+  if platform.is_huaweios?
+    pkg.url "http://buildsources.delivery.puppetlabs.net/nokogiri-precompiled-huaweios-#{pkg.get_version}.tar.gz"
```

Comment:
How about if I include a README file in the tarball describing the build process? Or do you think it should be commented in the component config here?